articles/how-to-manage-quantum-workspaces-with-the-azure-cli.md

```diff
@@ -53,18 +53,9 @@ In order to create a new Azure Quantum workspace, you'll need to know:
 - The resource group associated with the new workspace. (for example, **MyResourceGroup**).
 - An storage account on the same resource group and subscription than the quantum workspace. It's possible to [create a new storage account from the Az CLI tool](https://docs.microsoft.com/cli/azure/storage/account?view=azure-cli-latest&preserve-view=true#az_storage_account_create). (for example, **MyStorageAccount**)
 - The name of the quantum workspace to create. (for example, **MyQuantumWorkspace**)
+- The list of Azure Quantum providers and their SKUs to use in the workspace.
```

Comment:
Maybe we should add a note explaining what an SKU is, since some users won't be familiar with the term.

Reply:
Sounds good, let me add a note.